packages/console/src/ds-components/Select/index.tsx

```diff
@@ -108,7 +164,31 @@ function Select<T extends string>({
           setIsOpen(false);
         }}
       >
-        {options.map(({ value, title }) => (
+        {isSearchEnabled && isOpen && (
+          <div
+            className={styles.searchInputContainer}
+            style={{
+              position: 'fixed',
+              ...getSearchInputContainerStyles(),
```

Comment:
@xiaoyijun I'm not sure why the `fixed` position is needed in the dropdown. Do you have any context?

Reply:
@charIeszhao I used the 'fixed' position here because I wanted to position the input element above the select box, creating the impression that the input element is inside the select box. I could have placed the input element directly inside the select box, but that would have prevented users from selecting text inside the input with their mouse because of the overlay from the dropdown.

Reply:
Hmmm, I see. Let me see if there's a better solution for this. For now it's OK